src/cryptoadvance/specter/device.py

```diff
@@ -11,6 +11,10 @@ def __init__(self, name, alias, device_type, keys, fullpath, manager):
         self.fullpath = fullpath
         self.manager = manager
 
+    def create_psbts(self, base64_psbt, wallet):
```

Comment:
Works :) But maybe if no method is available, we should show the PSBT/ a button to copy it and tell the user to use the paste transaction button to do the signing? Right now it just brings a popup with no options. You can just take the copy PSBT functionality from the Raw PSBT section in the transaction details.

Reply:
That's the next thing I was thinking about. I thought about adding a "Generic wallet" option that would include QR code with base64 psbt, copy-text and download file - all with the same base64 psbt. Especially taking into account that "Other" option is not available anymore in the import device screen - we show an error if the type is not selected.

Reply:
That sounds even better :)